src/status_im/transport/handlers.cljs

```diff
@@ -171,6 +171,8 @@
                           (keep :js-obj)
                           (apply array))]
      (when (pos? (.-length js-messages))
-       (.. web3
-           -shh
-           (confirmMessagesProcessed js-messages (fn [_ _])))))))
+       (.confirmMessagesProcessed (transport.utils/shh web3)
+                                  js-messages
+                                  (fn [err resp]
+                                    (when err
+                                      (log/info "Confirming message processed failed"))))))))
```

Comment:
This is ok to be `log/info` and not `log/error`? 

Seem info is ok with me, we don't have to notify the user for something like that, just asking to make sure.

Reply:
Confirming messages is just perf tweak, something app can perfectly work without, so I think `info/warn` level is appropriate.